catalyst/contrib/dl/callbacks/__init__.py

```diff
@@ -24,6 +26,24 @@
         )
         raise ex
 
+try:
+    import kornia
+    from .kornia_transform import BatchTransformCallback
+except ImportError as ex:
+    if settings.cv_required:
+        logger.warning(
+            "some of catalyst-cv dependencies not available,"
+            " to install dependencies, run `pip install catalyst[cv]`."
+        )
+        raise ex
+except UnsupportedNodeError as ex:
+    logger.warning(
+        "kornia has requirement torch>=1.5.0,"
+        " probably you have an old version of torch which is incompatible."
```

Comment:
could we also add " to install dependencies, run `torch>=1.5.0`."?

Reply:
Yes, we could,
but wouldn't it lead to overwhelming of log message with information?

Reply:
why? I think it could be useful `copy-paste` message to handle warning